### ISSUE TITLE
Rubberduck Add-In could not be loaded: System.IO.FileLoadException

### ISSUE THREAD

**Issue by cswartzvi** [rubberduck-vba/Rubberduck#4640]
I tried to upgrade Rubberduck from `v2.3.0.4221` (which worked splendidly) to `v2.3.1.4308` and was greeted with the following error upon opening the VBE:

![rubberduck-error](https://user-images.githubusercontent.com/1111348/50020588-6b04db80-ffa4-11e8-9595-5360f93504f9.png)

System:
- Windows 10 (Version 1803, Build: 17134.471)
- Office 2016 

The corresponding log file is attached:
[RubberduckLog.txt](https://github.com/rubberduck-vba/Rubberduck/files/2681303/RubberduckLog.txt)

Notes:
- I installed this version, and the past version, without admin rights (I do not have admin rights at work)
- A number of my work colleagues have installed Rubberduck (also w/o admin rights) on similar systems and have not seen this error (yet)
- I tried downgrading to `v2.3.0.4221` but the problem persists. Perhaps it is a settings issue?
- I noticed that the Add-In is, apparently, looking for office version 12.0. Isn't that odd, given I am using Office 2016 (Microsoft Office 16.0 Object Library)?
- Most peculiar of all… The log mentions the folder `C:\projects` which does not exist. Strangely enough I first encountered this error while tying to open a file in the parent folder `D:\projects` but I have since reproduced this error on other files and blank Excel spreadsheets and Word documents.

Any help or direction you could provide would be most appreciated. Thanks!

**Comment by bclothier** [comment]
Wait a minu
The fact that access was denied makes me think that there is some external factors involved. So we need to follow up with some questions

1) Was the workbook read-only, or on a read-only media?
2) Was the file in a trusted location? 
3) Did you try to open the VBE without enabling the contents? 
4) Can you reproduce with a different or a blank workbook?
5) Given that you are the only one to experience this, can you compare the add-ins you have installed (both Excel and VBA) between you and your colleagues and test without them loaded?

**Comment by cswartzvi** [comment]
@retailcoder :
> The assembly pubic key tokens aren't sensitive at all, they're just public IDs for known assemblies, namely Office.dll v12.0 in this case (key tokens are identical for that assembly, on every machine in the world)

Haha, thanks for pointing that out. Old habit dies hard I guess; we're a little overly security conscious at my work.

@bclothier :

> 1. Was the workbook read-only, or on a read-only media?

Nope

> 2. Was the file in a trusted location?

I have not altered Excel's trusted locations (I confirmed they were the same as a colleague's) and the original file was in active development for months in the same location.

> 3. Did you try to open the VBE without enabling the contents?

I do not recall, but I don't believe so.

> 4. Can you reproduce with a different or a blank workbook?

Yes, I also encounter the same error trying to open the VBE on a blank Excel spreadsheet or a blank Word document (this was definitely not the case before).

> 5. Given that you are the only one to experience this, can you compare the add-ins you have installed (both Excel and VBA) between you and your colleagues and test without them loaded?

Here's the interesting part, I just had my colleague upgrade to `v2.3.1.4308` and everything works fine. I tested his installation with and without our additional Excel / VBE add-ins loaded (MZ-Tools and in-house add-ins). Smells like a configuration issue, right?

**Comment by retailcoder** [comment]
My fear is actually a borked Office install... are you in a position to repair the Office installation?

**Comment by cswartzvi** [comment]
No, I am not. However, I might be able to persuade those that are in that position (as a last resort). 
Fortunately, MZ-Tools loads (both at start-up and manually) and runs normally.

**Comment by retailcoder** [comment]
MZ-Tools is using its own interop libraries.. this complicates the comparison.

You wouldn't happen to have multiple, side-by-side installs of Office?

**Comment by retailcoder** [comment]
If 2010 was installed on top of 2016, then the registrations are likely broken... but then that doesn't explain how RD worked before the update (with broken registrations RD wouldn't have worked before)... note that non-chronologically installed side-by-side Office installs aren't a Microsoft-supported scenario either (and chronological side-by-side installs *should* work, but are discouraged).
